pkg/labels/filter.go

```diff
@@ -159,18 +159,16 @@ func defaultLabelPrefixCfg() *labelPrefixCfg {
 	}
 
 	expressions := []string{
-		k8sConst.PodNamespaceLabel,                                   // include io.kubernetes.pod.namspace
-		k8sConst.PodNamespaceMetaLabels,                              // include all namespace labels
-		"!io.kubernetes",                                             // ignore all other io.kubernetes labels
-		"!.*kubernetes.io",                                           // ignore all other kubernetes.io labels (annotation.*.k8s.io)
-		"!pod-template-generation",                                   // ignore pod-template-generation
-		"!pod-template-hash",                                         // ignore pod-template-hash
-		"!controller-revision-hash",                                  // ignore controller-revision-hash
-		"!annotation." + k8sConst.CiliumK8sAnnotationPrefix,          // ignore all cilium annotations
```

Comment:
I think we have to schedule removal of this to later or we risk changing security identity for existing pods. There is no code actually removing the annotation so an upgraded cluster will still have these annotations on.

Reply:
:man_facepalming: I thought those were to filter-in and not filter-out